model/group_status.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+// GroupStatus represents the status of a group.
+type GroupStatus struct {
+	InstallationsCount           int64
+	InstallationsRolledOut       int64
+	InstallationsAwaitingRollOut int64
+}
```

Comment:
Only 1/5 on this, but maybe it would be clearer if this was something like

```suggestion
type GroupStatus struct {
	InstallationsTotal          int64
	InstallationsUpdated        int64
	InstallationsRequiringUpdate int64
}
```
The only reason I suggest it is `rolling` normally just refers to the installations actively being updated at any one moment.

Reply:
👍  I would just change `InstallationsRequiringUpdate` to `InstallationsAwaitingUpdate`.